x11-packages/qbittorrent/build.sh

```diff
@@ -1,20 +1,20 @@
 TERMUX_PKG_HOMEPAGE=https://www.qbittorrent.org/
-TERMUX_PKG_DESCRIPTION="A Qt based BitTorrent client"
+TERMUX_PKG_DESCRIPTION="A Qt6 based BitTorrent client"
 TERMUX_PKG_LICENSE="GPL-2.0, GPL-3.0"
 TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
 TERMUX_PKG_VERSION="5.0.0"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/qbittorrent/qBittorrent/archive/refs/tags/release-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=9a24a6b95e9afac826295b8203685a515b13e77eb98bb8ed21c814916b999f6e
-TERMUX_PKG_FORCE_CMAKE=true
-TERMUX_PKG_DEPENDS="libc++, libtorrent-rasterbar, openssl, qt6-qtbase, zlib"
-TERMUX_PKG_BUILD_DEPENDS="qt6-qttools, qt6-qtsvg, boost"
+TERMUX_PKG_BUILD_DEPENDS="qt6-qtsvg, qt6-qttools, boost"
+TERMUX_PKG_DEPENDS="libc++, libtorrent-rasterbar, openssl, python, qt6-qtbase, zlib"
```

Comment:
Should not python be in suggested package list?

Reply:
I'd do that, but there is no `TERMUX_SUBPKG_RECOMMENDS` so `qbittorrent-nox` couldn't also recommend it.

Reply:
```
$ grep -r RECOMMEND --include "*subpackage.sh"
packages/xcb-proto/python-xcbgen.subpackage.sh:TERMUX_SUBPKG_RECOMMENDS="python"
```

🤔

Reply:
Forgot to remove python in TERMUX_SUBPKG_DEPENDS.

Reply:
Yep just noticed, already pushing up a fixup.

Reply:
GH Actions seem to be stalled right now, that's not great.